api/decorators/Validators.ts

```diff
@@ -145,7 +145,7 @@ export function IsValidFeedbackType(validationOptions?: ValidationOptions) {
 @ValidatorConstraint()
 class FeedbackStatusValidator implements ValidatorConstraintInterface {
   validate(feedbackStatus: FeedbackStatus): boolean {
-    return Object.values(FeedbackStatus).includes(feedbackStatus) && feedbackStatus !== FeedbackStatus.SUBMITTED;
+    return Object.values(FeedbackStatus).includes(feedbackStatus);
```

Comment:
Not sure if we still need this, but run it and double check on your side

Reply:
Seems to work fine without it -- good catch